src/Resources/app/administration/src/module/swag-migration/acl/index.ts

```diff
@@ -0,0 +1,81 @@
+const MIGRATION_ACL_KEY = 'swag_migration';
+
+const acl = {
+    category: 'permissions',
+    parent: 'settings',
+    key: MIGRATION_ACL_KEY,
+    roles: {
+        viewer: {
+            privileges: [
+                'swag_migration_connection:read',
+                'swag_migration_data:read',
+                'swag_migration_fix:read',
+                'swag_migration_general_setting:read',
+                'swag_migration_logging:read',
+                'swag_migration_mapping:read',
+                'swag_migration_media_file:read',
+                'swag_migration_run:read',
+                'swag_migration_history:read',
+                'user_config:read',
+                'system_config:read',
```

Comment:
What's the use case of these? Would be nice if you could add a short PR description for your change 🙂